### PR TITLE
Add inventory item junction table wrapper and unit tests

### PR DESCRIPTION
**Pull request by sjimenez814**

### CODE REVIEW

**Review by dwr2118 — APPROVED** [review]
Looks good to go

File: src/main/java/service/controller/ItemsRouteController.java [hunk 1]
```diff
@@ -325,10 +326,6 @@ public ResponseEntity<String> getInventoryIdFromItemId(
       }
       Item item = itemList.get(0);
       UUID inventoryId = item.getInventoryId();
-      if (inventoryId == null) {
-        return new ResponseEntity<>(
-            "No inventory id found for item: " + item.getItemName(), HttpStatus.NOT_FOUND);
-      }
```
Comment (CarrilloMario831) [inline src/main/java/service/controller/ItemsRouteController.java:331]: I think this is helpful to keep

**Review by CarrilloMario831 — COMMENTED** [review]
Looks good. Just the one comment I had.

**Comment by dwr2118** [comment]
> Looks good. Just the one comment I had.

@CarrilloMario831 we don't need to check if inventoryId is null since the new data schema requires each item to be associated with an inventory. Therefore, there must always be an inventoryId for every item within the DB

**Review by CarrilloMario831 — APPROVED** [review]

**Pull request merged**